package.json

```diff
@@ -1,57 +1,57 @@
 {
-  "name": "Hello-Books-BE",
```

Comment:
I can't see any change in this file. What did you delete what was there before and added it back

Reply:
The update on this file was on the test script

Reply:
Treated accordingly